File: src/zig/number-9.zig
const std = @import("std");

pub fn main() !void {
    const num = std.crypto.random.intRangeAtMost(u8, 0, 9);

    if (num == 9) {
        std.debug.print("Number 9...\n", .{});
    } else {
        std.debug.print("Try again.\n", .{});
    }
}

File: src/java/number-9.java
import java.util.Random;

public class number_9 {
    public static void main(String[] args) {
        Random random = new Random();
        if (random.nextInt(10) == 9) {
            System.out.println("Number 9...");
        } else {
            System.out.println("Try again.");
        }
    }
}
